File: packages/pensopay_android/android/src/main/kotlin/com/flutter/pensopay/networking/pensopayapi/pensopaylink/models/PPAddress.kt
package com.flutter.pensopay.networking.pensopayapi.pensopaylink.models

import org.json.JSONObject

class PPAddress: JSONObject() {

    // Optional Properties

    var name: String? = null
    var att: String? = null
    var street: String? = null
    var city: String? = null
    var zip_code: String? = null
    var region: String? = null
    var country_code: String? = null
    var vat_no: String? = null
    var company_name: String? = null
    var house_number: String? = null
    var house_extension: String? = null
    var phone_number: String? = null
    var mobile_number: String? = null
    var email: String? = null

}

File: packages/pensopay_ios/ios/Classes/Networking/Pensopay API/Pensopay Link/Models/PPAddress.swift
//
//  PPAddress.swift
//  PensopaySDK
//
//  Created on 07/26/2022
//  Copyright © 2022 Pensopay. All rights reserved.
//

import Foundation

public class PPAddress : Codable {

    // MARK: - Properties
    
    public var name: String?
    public var att: String?
    public var street: String?
    public var city: String?
    public var zip_code: String?
    public var region: String?
    public var country_code: String?
    public var vat_no: String?
    public var company_name: String?
    public var house_number: String?
    public var house_extension: String?
    public var phone_number: String?
    public var mobile_number: String?
    public var email: String?
    

    // MARK: Init
    
    public init() {
        
    }
    
}
